### PR TITLE
PMM-5723 Reviewdog checks.

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

**Comment by it-percona** [comment]
[CLA assistant check](https://cla-assistant.percona.com:443/percona/postgres_exporter?pullRequest=20) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.percona.com:443/percona/postgres_exporter?pullRequest=20) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.percona.com:443/check/percona/postgres_exporter?pullRequest=20) it.</sub>

**Comment by BupycHuk** [comment]
@askomorokhov good point, could you do it, I'm PTO today

File: .golangci.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+linters-settings:
+  lll:
+    line-length: 140
+    tab-width: 4
+
+  unused:
+    check-exported: true
```
Comment (percona-csalguero) [inline .golangci.yml:8]: Wouldn't this generate too many false positives?

Comment (BupycHuk) [reply .golangci.yml:8]: shouldn't, so it's config for all linters including optional ones.

**Pull request merged**